inc/admin.php

```diff
@@ -275,6 +275,7 @@ function enqueue_admin_scripts( $prefix = '', $force = false ) {
 				'directory_enabled'         => ! empty( $directory_enabled ),
 				'tutorials_enabled'         => ! empty( $tutorials_enabled ),
 				'copy_content'              => siteorigin_panels_setting( 'copy-content' ),
+				'cache'						=> array(),
```

Comment:
Shouldn't this be something like `get_transient( 'siteorigin_panels_tutorials' )` ?